### DISCUSSION TITLE
Css not working

### DISCUSSION THREAD

**Discussion by Shresth-Sharma** [community/community#22367] [Other Feature Feedback, Questions, & Ideas]
<p>i am not able to use css on github pages i am uploading a file named style.css and when i open it in github pages and go to inspect i get no css code there and css is not working <a href="https://shresth-sharma.github.io/kuhu1/" class="inline-onebox" rel="noopener nofollow ugc">Sanskriti Sharma</a></p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The spelling is different: Your file is called <code>style.css</code> in all lower case, but the <code>index.html</code> file includes <code>Style.css</code> with a capital <code>S</code> at the start.</p>
<aside class="onebox githubblob" data-onebox-src="https://github.com/Shresth-Sharma/kuhu1/blob/18da9f74a7236532486a91d6f83b0f05988a6700/index.html#L5">
  <header class="source">

      <a href="https://github.com/Shresth-Sharma/kuhu1/blob/18da9f74a7236532486a91d6f83b0f05988a6700/index.html#L5" target="_blank" rel="noopener">github.com</a>
  </header>

  <article class="onebox-body">
    <h4><a href="https://github.com/Shresth-Sharma/kuhu1/blob/18da9f74a7236532486a91d6f83b0f05988a6700/index.html#L5" target="_blank" rel="noopener">Shresth-Sharma/kuhu1/blob/18da9f74a7236532486a91d6f83b0f05988a6700/index.html#L5</a></h4>



  <pre class="onebox">    <code class="lang-html">
      <ol class="start lines" start="1" style="counter-reset: li-counter 0 ;">
          <li>&lt;!DOCTYPE html&gt;</li>
          <li>&lt;html&gt;</li>
          <li> &lt;head&gt;</li>
          <li>  &lt;title&gt;Sanskriti Sharma&lt;/title&gt;</li>
          <li class="selected">  &lt;link rel = "styleSheet" type="text/css" href ="Style.css"/&gt;</li>
          <li> &lt;/head&gt;</li>
          <li> &lt;body&gt;</li>
          <li>  &lt;div class="kuhu1"&gt;</li>
          <li>    &lt;img src="b.png" Width=300 Height=300/&gt;</li>
          <li>    &lt;img src="kahu1.jpeg" Width=300 Height=300 /&gt;</li>
          <li>    &lt;img src="b.png" Width=300 Height=300/&gt;</li>
          <li>  &lt;/div&gt;</li>
          <li>  &lt;div class="kuhu1"&gt;</li>
          <li>    &lt;H1&gt;Have a fabulous birthday!&lt;/H1&gt;</li>
          <li>  &lt;/div&gt;</li>
      </ol>
    </code>
  </pre>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Make sure the capitalization is the same and it should work. :smiley_cat:</p>

**Comment by Shresth-Sharma** [comment]
<p>Thank You Very Much ,It’s working now</p>
